test/parallel/test-tls-pfx-gh-5100-regr.js

```diff
@@ -19,8 +19,7 @@ const server = tls.createServer({
 }, common.mustCall(function(c) {
   assert.strictEqual(
     c.authorizationError,
-    null,
-    'authorizationError must be null'
+    null
   );
```

Comment:
I think this could be reformatted to be on one line like:

```js
assert.strictEqual(c.authorizationError, null);
```

If you could push a commit to your branch that makes this change then that would be great.